### ISSUE TITLE
How do you use interpolation workflow

### ISSUE THREAD

**Issue by Vektor369** [logtd/ComfyUI-LTXTricks#4]
I'm looking at the interpolation workflow, and it seems it's generating the first and last images when it's run? How would I go about using two images I already have? And what's the Load Video node for at the top left?

**Comment by logtd** [comment]
The interpolation workflow is supposed to interpolate between a few frames of something like a video. So to give an example of that I loaded the first and last frames of a video and compared to the input video to see how the model performed.

I'm not sure what your two images are, but it's not intended to be used to interpolate between two drastically different images and make something coherent.

You can change it to load separate images if you want.

**Comment by Vektor369** [comment]
Yup. That's exactly what I have. Two similar images as though the first and last frames of a shot from a video.
That's what I'm asking: How do I change it to load separate images. What do I plug the Load Image nodes into, and what do I take out?
Thanks.

**Comment by logtd** [comment]
On the setup on the left there should be two nodes that select frame indexes from the video upload node. Those then go VAE Encode and then the latents go to `Add LTX Latent Guide`. Make the Load Image nodes go to VAE Encode nodes
